acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/steps/ContentSteps.java

```diff
@@ -324,4 +330,49 @@ public void allOtherDetailsAreHidden() throws Throwable {
             consumablePublicationPage.getDisplayedElementsThatShouldBeHiddenForUpcomingPublication(), is(empty())
         );
     }
+
+    @Given("^I have a released publication with nominal date falling before (\\d+) weeks from now$")
+    public void iHaveAReleasedPublicationWithNominalDateFallingBeforeWeeksFromNow(final int weeksFromNow)
+        throws Throwable {
+
+        final Publication publicationWithNominalDateBeforeCutOff = TestDataFactory.createPublicationWithNoAttachments()
+            .withNominalDate(Instant.now()
+                .plus(DAYS_IN_WEEK * weeksFromNow, ChronoUnit.DAYS))
+            .build();
+
+        testDataRepo.setCurrentPublication(publicationWithNominalDateBeforeCutOff);
+
+        testDataSteps.createPublishedPublication(publicationWithNominalDateBeforeCutOff);
+    }
+
+    @Given("^I have a released publication with nominal date falling after (\\d+) weeks from now$")
+    public void iHaveAReleasedPublicationWithNominalDateFallingAfterWeeksFromNow(final int weeksFromNow)
+        throws Throwable {
+
+        final Publication publicationWithNominalDateBeforeCutOff = TestDataFactory.createPublicationWithNoAttachments()
+            .withNominalDate(Instant.now()
+                .plus(DAYS_IN_WEEK * weeksFromNow, ChronoUnit.DAYS)
+                .plus(1, ChronoUnit.DAYS))
+            .build();
+
+        testDataRepo.setCurrentPublication(publicationWithNominalDateBeforeCutOff);
+
+        testDataSteps.createPublishedPublication(publicationWithNominalDateBeforeCutOff);
+    }
+
+    @Then("^Nominal Publication Date is displayed using format \"([^\"]*)\"$")
+    public void nominalPublicationDateIsDisplayedUsingFormat(final String dateFormat) throws Throwable {
+
+        final Publication.NominalPublicationDate nominalPublicationDate =
+            testDataRepo.getCurrentPublication().getNominalPublicationDateDate();
+
+        // Obviously, this way of format resolution is brittle but it was chosen for simplicity as there is no need
+        // for anything more elaborate at the moment.
+        final String expectedDate = "1 Jan 2000".equals(dateFormat)
```

Comment:
This would get a lot cleaner if we used a mockable time provider

Reply:
The above comment about the reasons for not introducing one in context of end-to-end acceptance tests executed against a standalone application apply here as well.

Following our discussion I'm leaving it as is.

Reply:
Agreed.